util/containers/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# syntax=docker/dockerfile:1.4
+
+ARG NODE_VERSION
+ARG RUST_VERSION
+
+
+FROM --platform=${BUILDPLATFORM} docker.io/node:${NODE_VERSION}-alpine AS frontend-build
+
+WORKDIR /build
+COPY frontend/package.json frontend/package-lock.json /build/
+RUN npm i
+
+COPY frontend .
+RUN npm ci
+RUN npx relay-compiler
+RUN npx webpack --progress --mode=production
```

Comment:
Is `--progress` well behaved inside `docker build`? Or does it screw up the terminal output? In the latter case, I would just remove it.